code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -257,12 +270,18 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/styptic_powder/overdose_process(mob/living/M)
+	M.adjustBruteLoss(4, 0)
+	..()
+	. = 1
+
 /datum/reagent/medicine/salglu_solution
 	name = "Saline-Glucose Solution"
 	id = "salglu_solution"
 	description = "Has a 33% chance per metabolism cycle to heal brute and burn damage.  Can be used as a blood substitute on an IV drip."
 	reagent_state = LIQUID
 	color = "#DCDCDC"
+	overdose_threshold = 0
```

Comment:
it never overdoses, rather. 

logic is `if(overdose_threshhold && amount > overdose_threshhold)` meaning that a threshhold <= 0 disables overdosing completely.